dotnet-desktop-guide/net/wpf/documents/how-to-display-print-dialog.md

```diff
@@ -1,28 +1,54 @@
 ---
 title: How to display a print dialog
-description: 
-ms.date: 
+description: Learn how to print from your application by using the System.Windows.Controls.PrintDialog class to open a standard Microsoft Windows print dialog box.
+ms.date: 08/20/2021
 dev_langs: 
   - "csharp"
   - "vb"
 helpviewer_keywords: 
-  - 
+  - "invoking print dialogs [WPF]"
+  - "print dialogs [WPF], invoking"
 ---
 
 # How to display a print dialog box (WPF .NET)
 
-<!--
+Want to print from your application? You can use the <xref:System.Windows.Controls.PrintDialog> class to open a standard Microsoft Windows print dialog box. Here's how.
 
-Source material for this article:
-- https://docs.microsoft.com/en-us/dotnet/desktop/wpf/advanced/how-to-invoke-a-print-dialog?view=netframeworkdesktop-4.8
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
 
-Notes:
-- None
+## Examples
 
--->
+The <xref:System.Windows.Controls.PrintDialog> class provides a single control for print configuration and XPS job submission. The control is easy to use and can be instantiated by using XAML markup or code. These examples create and display a `PrintDialog` instance using code.
 
-!!INTRO
+### Print the whole document
```

Comment:
We could promote these sections to ## to make them appear in the TOC. Then, you could remove the ## Examples header.

Reply:
Done.